earthpy/io.py

```diff
@@ -268,6 +275,8 @@ def _download(self, url, path, kind, replace, verbose):
 
         if verbose is True:
             print("Downloading from {}".format(url))
+        else:
```

Comment:
ok so here @sgillies if i understand correctly, we are removing the conditional, and it will log regardless which is good. but what allows it to print a message out for the user? bare with me - this logging approach is new and i'm learning here a lot!!